source/Calamari.AzureAppService/Calamari.AzureAppService.csproj

```diff
@@ -17,19 +17,20 @@
     <TargetFramework>net6.0</TargetFramework>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Azure.Identity" Version="1.2.3" />
+    <PackageReference Include="Azure.Identity" Version="1.10.0" />
     <PackageReference Include="Azure.ResourceManager" Version="1.7.0" />
     <PackageReference Include="Azure.ResourceManager.AppService" Version="1.0.2" />
     <PackageReference Include="Azure.ResourceManager.ResourceGraph" Version="1.0.1" />
     <PackageReference Include="Microsoft.AspNetCore.WebUtilities" Version="2.2.0" />
     <PackageReference Include="Microsoft.Azure.Management.AppService.Fluent" Version="1.37.1" />
     <PackageReference Include="Microsoft.Azure.Management.Fluent" Version="1.37.1" />
-    <PackageReference Include="Microsoft.Azure.Management.Websites" Version="3.1.1" />
-    <PackageReference Include="Microsoft.Identity.Client" Version="4.48.1" />
+    <PackageReference Include="Microsoft.Azure.Management.Websites" Version="4.0.0" />
+    <PackageReference Include="Microsoft.Identity.Client" Version="4.56.0" />
     <PackageReference Include="Microsoft.DotNet.Analyzers.Compatibility" Version="0.2.12-alpha">
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
     </PackageReference>
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```

Comment:
Was there a specific reason for swapping to NewtonSoft?
Sometimes, serialisation libraries can give you different results if you don't setup the configuration right

Reply:
Calamari.CloudAccounts builds net452 and System.Text.Json is only available from net461 on. Newtonsoft is case insensitive by default, which matches the existing behaviour we specified in System.Text.Json in the `source/Calamari.AzureAppService/Behaviors/TargetDiscoveryBehaviour.cs`

Reply:
sorry, just saw your response from earlier